attributes/default.rb

```diff
@@ -527,4 +527,7 @@
 
 # Opensearch embedding db
 default['hopsworks']['opensearch']['default_embedding_index']     = node['hopsworks']['opensearch']['default_embedding_index'].to_s.empty? ? "" : node['hopsworks']['opensearch']['default_embedding_index']
-default['hopsworks']['opensearch']['num_default_embedding_index'] = node['hopsworks']['opensearch']['num_default_embedding_index'].to_s.empty? ? 1 : node['hopsworks']['opensearch']['num_default_embedding_index']
\ No newline at end of file
+default['hopsworks']['opensearch']['num_default_embedding_index'] = node['hopsworks']['opensearch']['num_default_embedding_index'].to_s.empty? ? 1 : node['hopsworks']['opensearch']['num_default_embedding_index']
+
+# enable conda install HWORKS-302
+default['hopsworks']['enable_conda_install'] = "true"
```

Comment:
You need to add this parameter to the `metadata.rb` file so that it can be overwritten in the cluster definition.

Reply:
fixed now